### PR TITLE
Feedback improvements + split livestream page 

### LINKED ISSUE #120: Improve instructions to include "prezidentská místnost"

### LINKED ISSUE #121: [place] recommend nearby restaurants
https://www.google.com/maps/place/Restaurace+U+Holi%C5%A1e/@50.1049226,14.4456406,17z/data=!3m1!4b1!4m5!3m4!1s0x470b94b3ed9c2693:0x3772031ac187930!8m2!3d50.1049226!4d14.4478293

### LINKED ISSUE #130: [training detail] link place to place detail, not map

### LINKED ISSUE #133: [video] Split Livestream page + tweet about it

### LINKED ISSUE #134: [place] add parking tips
"parkování před budovou bohužel není možné, hosté budou muset zaparkovat někde poblíž – v okolních ulicích (např. U Vody nebo Jankovcova, 250-500m chůze od budovy) či na hlídaných parkovištích cca 800m chůze od budovy (https://www.google.com/maps/place/Parking/@50.1083464,14.4418636,18.24z/data=!4m5!3m4!1s0x470b94b568f8602b:0xee1c935d971c7b45!8m2!3d50.1082527!4d14.4423303 nebo https://www.google.com/maps/place/P%2BR+Hole%C5%A1ovice/@50.1088325,14.441015,18.24z/data=!4m5!3m4!1s0x470b94caa6ae1b9b:0xcfc14e74ebc3b4c5!8m2!3d50.1089201!4d14.4411716).

 

Pokud bude před budovou volné místo pro návštěvy, může ho využít lektor, ale garantovat nebo rezervovat ho nemůžeme, děkuji za pochopení."

### LINKED ISSUE #136: [training] Add is_public
## Related

+ require description

image

### PR DESCRIPTION
**Pull request by TomasVotruba**
- Closes #133
- Closes #136
- Closes #135
- Closes #134
- Closes #121
- Closes #130 
- Closes #120

### CODE REVIEW

File: packages/Training/src/Repository/TrainingRepository.php [hunk 1]
```diff
@@ -33,14 +33,20 @@ public function fetchAll(): array
...
             ->setParameter(':weekAgo', DateTime::from('- 7 days'))
             ->setParameter(':nextWeek', DateTime::from('+ 7 days'))
+            // has at least one registration
+            ->join('tt.registrations', 'tr')
             ->addGroupBy('tt.startDateTime')
-            ->orderBy('tt.startDateTime') // put more recent first
+            // prefer closest to now - so people rating training at the day of training will get the date fo training :)
+            // see https://stackoverflow.com/questions/21490993/expected-known-function-got-timedifffunction-not-found-in-doctrine-orm
+            ->orderBy('time_diff(tt.startDateTime, CURRENT_TIMESTAMP())', 'DESC')
             ->getQuery()
```
Comment (TomasVotruba) [inline packages/Training/src/Repository/TrainingRepository.php:50]: @JanMikes You're gonna love this one :D I bet it breaks the web

**Comment by TomasVotruba** [comment]
Let's break things :ship:

**Comment by JanMikes** [comment]
Emergency line is ready ☎️ 🚒

**Comment by TomasVotruba** [comment]
Looks good, just one bug with that group issue. I'll try to add back original groupping

**Comment by TomasVotruba** [comment]
So the double groupping is already there :/

https://github.com/pehapkari/pehapkari.cz/blob/master/packages/Training/src/Repository/TrainingRepository.php#L46-L47

Could you look at it?

https://pehapkari.cz/feedback/

**Pull request merged**